java/test/jmri/jmrix/acela/AcelaTurnoutManagerTest.java

```diff
@@ -51,9 +51,18 @@ public void testAsAbstractFactory() {
             log.debug("by user name:   {}", t.getByUserName("my name"));
         }
 
-        Assert.assertTrue(null != t.getBySystemName("AT11"));
-        Assert.assertTrue(null != t.getByUserName("my name"));
+        Assert.assertNotNull(t.getBySystemName("AT11"));
+        Assert.assertNotNull(t.getByUserName("my name"));
+    }
 
+    @Test
+    @Override
+    public void testSetAndGetOutputInterval() { // AcelaTurnoutManager has direct access to Memo
```

Comment:
This test should be in the TurnoutManagerTestBase (or whatever the parent class of the TurnoutManager tests is named), and only be overridden for TurnoutManagers that don’t implement the interval change.

Reply:
I overrode the test in a few cases where ``memo`` or ``lnis`` was present, but that might not be very useful now I know how to access the memo in a standard way.